libs/tokens/src/hooks/tokens/useNetworkLogo.ts

```diff
@@ -16,7 +16,13 @@ export function useNetworkLogo(chainId?: number) {
       return undefined
     }
 
-    return theme.darkMode ? getChainInfo(chainId)?.logo?.dark : getChainInfo(chainId)?.logo?.light
+    const chainInfo = getChainInfo(chainId)
+
+    if (chainInfo?.name === 'arbitrum_one') {
```

Comment:
Can you compare chainId to SupportedChainId.ARBITRUM_ONE instead?
I'd prefer that to a string that might be harder to find/change down the line.